css/style.css

```diff
@@ -1,365 +1,392 @@
-/* Color Scheme. Please use var() to set colours. This will make site-wide changes later much easier */
-:root{
-    --primary-grey:#222831;
-    --seconday-grey:#393E46;
-    --accent-yellow: #FFD369;
-    --error-red: #C14848;
+/* USE var() TO SET COLOURS */
+:root {
+    --primary-color: #222831;
+    --secondary-color: #393E46;
+    --accent: #FFD369;
+    --error: #C14848;
 }
 
-
-
-/* general global settings */
-body{
-    background-color: var(--primary-grey);
+body {
+    background-color: var(--primary-color);
 }
-input:focus{
+
+input:focus {
     outline: none;
 }
 
+/* -- GENERAL STYLING, see Figma -- */
 
-
-
-/* General font styling (check figma file for notes on how to use!) */
-h1,h2,h3{
+h1,
+h2,
+h3 {
     font-family: 'Bungee Inline', cursive;
-    color: var(--accent-yellow);
```

Comment:
Keeping with the "global" theme, this color name was removed from the actual var, like the others. All referenced uses were updated so the name changes wouldn't break.

Reply:
Cool